src/Symfony/Bundle/FrameworkBundle/Command/RouterDebugCommand.php

```diff
@@ -56,6 +56,7 @@ protected function configure()
                 new InputOption('show-controllers', null, InputOption::VALUE_NONE, 'Show assigned controllers in overview'),
                 new InputOption('format', null, InputOption::VALUE_REQUIRED, 'The output format (txt, xml, json, or md)', 'txt'),
                 new InputOption('raw', null, InputOption::VALUE_NONE, 'To output raw route(s)'),
+                new InputOption('filter', null, InputOption::VALUE_REQUIRED, 'Filter the routes (text format only)'),
```

Comment:
Sadly this description doesn't explain the purpose of the option. What about using something like this?

```
Display only the routes whose names contain this text.
```